### PR TITLE
🎨 #14 create reusable project item component

### PR DESCRIPTION
**Pull request by jbdura**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Screenshot from 2023-08-23 12-58-06
Screenshot from 2023-08-23 12-57-40

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/data/sections/src/lib/data-sections.module.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
```
Comment (LemmyMwaura) [inline libs/data/sections/src/lib/data-sections.module.ts:1]: There's no need for this file. Our component has no declarations to manage.

File: libs/elements/cards/src/lib/elewa-project-item-card/elewa-project-item-card.component.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { ProjectItem } from '@elewa-website/models/sections/projects/cards'
+
+@Component({
+  selector: 'elewa-website-elewa-project-item-card',
+  templateUrl: './elewa-project-item-card.component.html',
+  styleUrls: ['./elewa-project-item-card.component.scss'],
+})
+export class ElewaProjectItemCardComponent implements OnInit {
+  
+  // Import of the interface
```
Comment (LemmyMwaura) [inline libs/elements/cards/src/lib/elewa-project-item-card/elewa-project-item-card.component.ts:11]: No need for this comment. Only use JsDoc comments to describe properties and methods. This allows us to get some good Intellisense.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Good job so far. Update the Pr description and also clean up the design a bit.

**Comment by jbdura** [comment]
Changes done

File: libs/data/sections/src/lib/ highlighted-projects.data.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { ProjectItem } from '@elewa-website/models/sections/projects/cards'
+// import { ProjectItem } from "../../models/sections/projects/cards/src";
```
Comment (LemmyMwaura) [inline libs/data/sections/src/lib/ highlighted-projects.data.ts:2]: remove this comment

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Kindly share an image of the updated design.

**Comment by LemmyMwaura** [comment]
Closing this as it's been resolved in #36

**Pull request closed without merging**